src/scripts/coveralls.sh

```diff
@@ -73,3 +73,10 @@ echo "Reporting coverage"
 set -x
 # shellcheck disable=SC2086
 ./coveralls report $args
+set +x
+
+exit_status=$?
+if [ "${COVERALLS_FAIL_ON_ERROR}" != "1" ]; then
+  exit_status=0
+fi
+exit "${exit_status}"
```

Comment:
I'm wondering whether it would be OK to just add this:

```suggestion
exit 0
```

That's what's done here:

https://github.com/academia-edu/coveralls-circleci-orb/blob/d1a1f075e55373fbe57c5ab810a87ebce41cd44c/src/scripts/coveralls.sh#L39